src/platform.ts

```diff
@@ -116,11 +116,17 @@ export class PowerShellExeFinder {
      * @param configuredPowerShellPath the PowerShell path configured by the user.
      */
     public fixWindowsPowerShellPath(configuredPowerShellPath: string): string {
+        const altWinPS = this.findWinPS({ useAlternateBitness: true });
+
+        if (!altWinPS) {
+            return configuredPowerShellPath;
+        }
+
+        const lowerAltWinPSPath = altWinPS.exePath.toLocaleLowerCase();
         const lowerConfiguredPath = configuredPowerShellPath.toLocaleLowerCase();
-        const lowerAltWinPSPath = this.alternateBitnessWinPS.exePath.toLocaleLowerCase();
 
         if (lowerConfiguredPath === lowerAltWinPSPath) {
-            return this.winPS.exePath;
+            return this.findWinPS().exePath;
```

Comment:
You need to run findWinPS here?

Reply:
So basically, if you've configured a PowerShell path and it points to a non-native bitness, this method is supposed to fix it.

`findWinPS()` gives us back the native bitness Win PS.

It doesn't actually touch the filesystem, and is even cached.

Now that the laziness has come out, the way to reliably get the cached value is to call the method.